### DISCUSSION TITLE
alist挂载s3桶时报签名失败

### DISCUSSION THREAD

**Discussion by yjiu1990** [AlistGo/alist#5004] [General]
我这个ak与sk的肯定是有权限的，给的是管理员权限
<img width="1024" alt="image" src="https://github.com/alist-org/alist/assets/41354304/3f684cc5-115c-4948-945f-3e01a9d6083e">
<img width="834" alt="image" src="https://github.com/alist-org/alist/assets/41354304/049eca4a-8b51-47f2-b6c3-820e55d8539f">

**Comment by anwen-anyi** [comment]
我这里没问题呢~
![image](https://github.com/alist-org/alist/assets/56105412/fc59fbec-3fec-4195-921d-7ba2805f8226)

**Comment by yjiu1990** [comment]
我这用本地，服务器2台使用docker搭建的，全部都是403，阿里云OSS是正常的，AWS就一直报403签名错误

**Comment by yjiu1990** [comment]
<img width="1133" alt="image" src="https://github.com/alist-org/alist/assets/41354304/a035aacb-0d3d-40f6-b0f6-5af81e5bae7d">

**Comment by yjiu1990** [comment]
我找到问题了，不知道是不是bug, 我第一次把地区还有Endpoint设置错误了，然后后面换成正确的，可能有缓存导致一直使用的信息是错误的，才会签名失败，然后我把地区改成错误的地区，再一次访问返回400，再改成正确的地区就正常了

**Reply by anwen-anyi** [reply]
> 我找到问题了，不知道是不是bug, 我第一次把地区还有Endpoint设置错误了，然后后面换成正确的，可能有缓存导致一直使用的信息是错误的，才会签名失败，然后我把地区改成错误的地区，再一次访问返回400，再改成正确的地区就正常了

不是bug，是缓存。在这个文件夹的上一级文件夹摁一下右下角的刷新就可以恢复了。或者重新添加一个也可以
